Makefile

```
@@ -38,3 +44,6 @@ scan:
 
 scan-%:
 	cd $(subst :,/,$*); syft scan . -o json | grype --fail-on low
+
+check-go-version-consistency:
+	hack/check_go_version.sh
```

```suggestion
	hack/check-go-version.sh
```

For consistency let's name files w/ `-`, and reserve `_` for when it is required (like w/ `_test.go`)